manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/persistence/models/Processor.java

```diff
@@ -17,6 +17,8 @@
 import com.redhat.service.smartevents.infra.v2.api.models.processors.ProcessorDefinition;
 
 @NamedQueries({
+        @NamedQuery(name = "PROCESSOR_V2.findByIdWithConditions",
+                query = "from Processor_V2 b left join fetch b.conditions where b.id=:id"),
```

Comment:
Could you be consistent oh "Copy & Paste Gods" and use `p` for the `Processor` alias?